test/data/test_multi_embedding_tensor.py

```diff
@@ -117,3 +118,130 @@ def test_clone():
     assert met_clone.values[0, 0] != 12345.
     met.offset[0] = -1
     assert met_clone.offset[0] != -1
+
+
+def test_cat():
+    # case: dim=0
+    tensor_list1 = [
+        torch.tensor([[0, 1, 2], [6, 7, 8]]),
+        torch.tensor([[3, 4], [9, 10]]),
+        torch.tensor([[5], [11]]),
+    ]
+    met1 = MultiEmbeddingTensor.from_tensor_list(tensor_list1)
+    tensor_list2 = [
+        torch.tensor([[12, 13, 14]]),
+        torch.tensor([[15, 16]]),
+        torch.tensor([[17]]),
+    ]
+    met2 = MultiEmbeddingTensor.from_tensor_list(tensor_list2)
+    for xs in [[met1, met2], (met1, met2)]:
```

Comment:
Testing both `list`/`tuple` as requested in https://github.com/pyg-team/pytorch-frame/pull/181#discussion_r1383949747.